docs-src/tcp.md

```diff
@@ -0,0 +1,23 @@
+Service APIs is designed to work with multiple protocols (unlike the HTTP only
+Ingress/v1 resource). [TCPRoute](spec/#networking.x-k8s.io/v1alpha1.TCPRoute) is
+one such route which allows for managing TCP traffic.
```

Comment:
I don't think we necessarily need the comparison to Ingress here. Though this is a great reminder that we should probably have an entire page devoted to the differences between these APIs and Ingress and/or some examples of how Ingress config could be translated.

Reply:
Removed the mention of Ingress.